engine.py

```diff
@@ -955,35 +956,29 @@ def _define_qt_base(self):
 
         :returns: dict
         """
-        base = {}
-        from PySide import QtCore, QtGui
-        base["qt_core"] = QtCore
-        base["qt_gui"] = QtGui
-        base["dialog_base"] = QtGui.QDialog
-
-        # tell QT to handle text strings as utf-8 by default
-        utf8 = QtCore.QTextCodec.codecForName("utf-8")
```

Comment:
@josh-t i know this is an old PR, but i am currently looking at https://github.com/shotgunsoftware/tk-shotgun/pull/6 which is essentially the same change but for the shotgun engine, and i noticed that we are removing the utf-8 cstring encoding. Does this not mean that unicode characters will be displayed incorrectly in the UIs?